comp/prizes/ceremony.yaml

```diff
@@ -34,10 +34,19 @@ description: >-
 
 
   Required actions:
-   - [ ] Get branded envelopes & cards for the prize ceremony
-   - [ ] Arrange a speaker for the prize ceremony
-   - [ ] Plan out the prize-giving ceremony
-   - [ ] Have a rehearsal of the prize-giving ceremony
+    - [ ] Get branded envelopes & cards for the prize ceremony
+    - [ ] Arrange a speaker for the prize ceremony
+    - [ ] Plan out the prize-giving ceremony
+    - [ ] Write a script for the prize-giving ceremony
+    - [ ] Reivew in the weeks before the competition:
```

Comment:
Thought: Should we explicitly add head of match ops in here too? Making sure the order looks sensible, schedule, operational plan etc. Since that's under match ops' remit.

Reply:
https://github.com/srobo/recurring-tasks/pull/124/commits/9281eab303a69c2f864190d3b7f3e3af0b81da3e